generators/ci-cd/templates/_.gitlab-ci.yml

```diff
@@ -82,10 +95,10 @@ before_script:
     - export MAVEN_USER_HOME=`pwd`/.maven
     <%_ if (!skipClient) { _%>
         <%_ if (clientPackageManager === 'yarn') { _%>
-    - ./mvnw com.github.eirslett:frontend-maven-plugin:install-node-and-yarn -DnodeVersion=v6.9.4 -DyarnVersion=v0.18.1
+    - ./mvnw com.github.eirslett:frontend-maven-plugin:install-node-and-yarn -DnodeVersion=v6.9.5 -DyarnVersion=v0.20.3
```

Comment:
Maybe update yarn.version in _pom.xml (node.version is already 6.9.5 in _pom.xml) and use default versions from _pom.xml?
Same for npm section and Jenkinsfile yarn and npm sections.